frontend/packages/db/supabase/migrations/20250409175500_add_resolved_columns_to_review_issue.sql

```diff
@@ -0,0 +1,12 @@
+-- Add resolvedAt and resolutionComment columns to ReviewIssue table
+BEGIN;
+
+-- Add resolvedAt column to ReviewIssue table
+ALTER TABLE "public"."ReviewIssue"
+ADD COLUMN "resolvedAt" timestamp(3) without time zone;
+
+-- Add resolutionComment column to ReviewIssue table
+ALTER TABLE "public"."ReviewIssue"
+ADD COLUMN "resolutionComment" text;
+
+END;
```

Comment:
Also, can be 

```sql
BEGIN;

ALTER TABLE "public"."ReviewIssue"
  ADD COLUMN "resolvedAt" timestamp(3) without time zone,
  ADD COLUMN "resolutionComment" text;

COMMIT;
```

Reply:
Thank you so much!✨
https://github.com/liam-hq/liam/pull/1275/commits/846eab52dbfa8d6fb081b8784420fe72e4a199dd